### PR TITLE
Import cleanup

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
This patch:
- removes some unused imports
- removes template auto import using `TwirlKeys.templateImports` setting: It makes templates dependencies hard to follow. Instead importing the necessary dependencies in each templates.

Example: `football` templates were automatically importing `pa._` which provides an implicit `StringToOption` function. It was not clear to me why passing a `String` to a fragment that was expecting an `Option[String]` argument was compiling successfully

I wanted to enable the `-Ywarn-unused-import` compiler option but it doesn't play well with Twirl templates: https://github.com/playframework/twirl/issues/105

## What is the value of this and can you measure success?
More explicit dependencies, less hidden magic

## Tested in CODE?
yes

### CODE REVIEW

File: project/Frontend.scala [hunk 1]
```diff
@@ -88,17 +87,10 @@ object Frontend extends Build with Prototypes {
     libraryDependencies ++= Seq(
       paClient,
       akkaContrib
-    ),
-    TwirlKeys.templateImports ++= Seq(
```
Comment (jfsoul) [inline project/Frontend.scala]: 😰

File: sport/app/football/views/matchStats/matchStatsPage.scala.html [hunk 2]
```diff
@@ -3,8 +3,7 @@
 @import implicits.Football._
 @import views.FootballHelpers._
 
-
-@(page: MatchPage, competition: Option[model.Competition])(implicit request: RequestHeader, context: model.ApplicationContext)
+@(page: _root_.football.controllers.MatchPage, competition: Option[model.Competition])(implicit request: RequestHeader, context: model.ApplicationContext)
```
Comment (jfsoul) [inline sport/app/football/views/matchStats/matchStatsPage.scala.html]: I'm not sure I understand why these have to be `_root_`, is there another `MatchPage` it might pick up otherwise?

**Review by jfsoul — APPROVED** [review]
Lgtm, just had one question

**Review by NataliaLKB — APPROVED** [review]
Yay to being more explicit 🎉 !

File: sport/app/football/views/matchStats/matchStatsPage.scala.html (hunk 2 above)
Comment (TBonnin) [inline sport/app/football/views/matchStats/matchStatsPage.scala.html]: You are right. It is not necessary.
However IntelliJ is confused and pick up the `football` package from another app (admin). That's why I originally added `_root_`. 
I removed it and will try to fix IntelliJ instead

Comment (jfsoul) [inline sport/app/football/views/matchStats/matchStatsPage.scala.html]: > will try to fix IntelliJ instead

Good luck, you may need it! 😄
